scranShipper/src/Display/RestaurantDisplay.jsx

```diff
@@ -0,0 +1,16 @@
+function RestaurantDisplay(){
+    const makeRestaurant = (restaurant) => {
+        return (<>
```

Comment:
Lines 3, 5, 9, and 12 have empty containers which are redundant because the functions are only returning one thing. Looks good otherwise.

Reply:
Having taken your comments into consideration, we found them to be only partially accurate, in that the containers on lines 9 and 12 were required for the good functioning of the function in question.